autogen/dns.tf

```diff
@@ -20,7 +20,7 @@
   Delete default kube-dns configmap
  *****************************************/
 resource "null_resource" "delete_default_kube_dns_configmap" {
-  count = local.custom_kube_dns_config || local.upstream_nameservers_config ? 1 : 0
+  count = (local.custom_kube_dns_config || local.upstream_nameservers_config) || var.skip_provisioners ? 1 : 0
```

Comment:
need to add explanation of this in skip_provisioners var description

Reply:
Updated description of the variable.

Reply:
This logic is wrong.

If you skip provisioners, no local-execs should *ever* be initialized.

Reply:
Fixed